src/commands/org/list/metadata-types.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import { Messages } from '@salesforce/core';
+import { DescribeMetadataResult } from 'jsforce/api/metadata';
+import { RegistryAccess } from '@salesforce/source-deploy-retrieve';
+import {
+  Flags,
+  loglevel,
+  orgApiVersionFlagWithDeprecations,
+  requiredOrgFlagWithDeprecations,
+  SfCommand,
+} from '@salesforce/sf-plugins-core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-org', 'metadata-types');
+
+export class DescribeMetadata extends SfCommand<DescribeMetadataResult> {
+  public static readonly summary = messages.getMessage('description');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+  public static readonly aliases = ['force:mdapi:describemetadata'];
+  public static readonly deprecateAliases = true;
+  public static readonly flags = {
+    'api-version': orgApiVersionFlagWithDeprecations,
+    loglevel,
+    'target-org': requiredOrgFlagWithDeprecations,
+    'output-file': Flags.file({
+      aliases: ['resultfile'],
+      deprecateAliases: true,
+      char: 'f',
+      description: messages.getMessage('flags.resultfile'),
```

Comment:
can we make the message key match the current flag name instead of the deprecated versions?